vnext/Microsoft.ReactNative/ReactInstanceSettingsBuilder.cpp

```diff
@@ -0,0 +1,50 @@
+#include "pch.h"
+#include "ReactInstanceSettingsBuilder.h"
+#include "ReactInstanceSettingsBuilder.g.cpp"
+#include "ReactInstanceSettings.h"
+
+namespace winrt::ReactNative {
+using namespace winrt::Microsoft::ReactNative;
+}
+
+namespace winrt::UI {
+using namespace winrt::Microsoft::UI;
+}
+
+namespace winrt::Microsoft::ReactNative::implementation {
+ReactInstanceSettingsBuilder::ReactInstanceSettingsBuilder() {
+  m_reactInstanceSettings = winrt::make<winrt::ReactNative::implementation::ReactInstanceSettings>();
+
+#if _DEBUG
+  m_reactInstanceSettings.UseDirectDebugger(true);
+  m_reactInstanceSettings.UseDeveloperSupport(true);
+#else
+  m_reactInstanceSettings.UseDirectDebugger(false);
+  m_reactInstanceSettings.UseDeveloperSupport(false);
+#endif
+}
+
+winrt::ReactNative::ReactInstanceSettings ReactInstanceSettingsBuilder::ReactInstanceSettings() {
+  return m_reactInstanceSettings;
+}
+
+winrt::ReactNative::ReactInstanceSettingsBuilder ReactInstanceSettingsBuilder::SetBundleRootPath(hstring const &path) {
+#if BUNDLE
```

Comment:
This BUNDLE flag is only defined in the app itself, so it can't be pre-built into the library. This should also be in the app code.

Reply:
Sure. I can remove the if check.